src/screens/SendConfirmationSheet.tsx

```diff
@@ -420,7 +420,7 @@ export const SendConfirmationSheet = () => {
     return existingAcct;
   }, [toAddress, userAccounts, watchedAccounts]);
 
-  let avatarName = removeFirstEmojiFromString(existingAccount?.label || contact?.nickname);
+  let avatarName = removeFirstEmojiFromString(contact?.nickname || existingAccount?.label);
```

Comment:
Okay similarly to the other instance of this, we were never showing contact nicknames for accounts with ens names since we always preferred ens names over contact nicknames.